File: python_programming/practices/financial_caculator.py
# SB 6th Financial Caculator

monthly_income = 4000
rent = 400
groceries = 300
utilities = 320
transportation = 480
savings = monthly_income * .2
print("my monthly income is", monthly_income)
print("my rent is", rent)
print("my grocery cost is", groceries)
print("my utility cost is", utilities)
print("my transportation cost is", transportation)
print("my rent is", rent, "which is",  rent / monthly_income*100, "percent of my income")
print("my grocery cost is", groceries, "which is",  groceries / monthly_income*100, "percent of my income")
print("my utility cost is", utilities, "which is",  utilities / monthly_income*100, "percent of my income")
print("my transportation cost is", transportation, "which is",  transportation / monthly_income*100, "percent of my income")
print("I save 20 percent of my income, which is", savings)
total = rent + utilities + transportation + groceries + savings
leftover = monthly_income - total
print("I have", leftover, "dollars to spend")

File: c_programming/notes/financialcaculator.c
// SB 6th financial caculator
#include <stdio.h>
#include <math.h>

int main(void) {
    float income = 4000.0;
    float rent = 100.0;
    float utilities = 70.0;
    float groceries = 150.0;
    float transportation = 350.0;
    float rentpercent = (rent / income)*100;
    float utilitiespercent = (utilities/income)*100;
    float grocerypercent = (groceries/income)*100;
    float transportpercent = (transportation/income) * 100;
    float moneysaved = (income*.2);
    float total = (rent+utilities+transportation+groceries+moneysaved);
    float leftover = (income-total);
    printf("Your rent is $100 and that's: %.1f percent of your income.", rentpercent);
    printf("Your utililties cost $70 and thats: %.1f percent of your income", utilitiespercent);
    printf("Your groceries cost $150 and thats: %.1f percent of your income", grocerypercent);
    printf("Your transportation costs $350 and thats: %.1f percent of your income", transportpercent);
    printf("I save: 20 percent of my income which is: %.1f", moneysaved);
    printf("My leftover money is: %.1f", leftover);
    return 0;
}

